dev-db/redict/redict-7.3.0.ebuild

```diff
@@ -0,0 +1,163 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# N.B.: It is no clue in porting to Lua eclasses, as upstream have deviated
+# too far from vanilla Lua, adding their own APIs like lua_enablereadonlytable
+
+inherit edo multiprocessing systemd tmpfiles toolchain-funcs
+
+if [ ${PV} = "9999" ]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://codeberg.org/redict/redict"
+else
+	SRC_URI="https://codeberg.org/redict/redict/archive/${PV/_/-}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~loong ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux"
+fi
+
+S=${WORKDIR}/${PN}
+
+DESCRIPTION="A persistent caching system, key-value, and data structures database"
+HOMEPAGE="https://redict.io"
+
+LICENSE="BSD Boost-1.0 GPL-3"
+SLOT="0/$(ver_cut 1-2)"
+IUSE="+jemalloc selinux ssl systemd tcmalloc test"
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="?? ( jemalloc tcmalloc )"
+
+COMMON_DEPEND="
+	jemalloc? ( >=dev-libs/jemalloc-5.1:= )
+	ssl? ( dev-libs/openssl:0= )
+	systemd? ( sys-apps/systemd:= )
+	tcmalloc? ( dev-util/google-perftools )
+"
+
+RDEPEND="
+	${COMMON_DEPEND}
+	acct-group/redis
+	acct-user/redis
```

Comment:
redict aims to be a drop-in replacement for redis.

on the migration guide, they recommend renaming the redis user and group to redict, or creating a new redict user and group with the same uid and gid as redis

we could also, instead, add a new redict user and group, with a new uid/gid, and notify the user to move the database permissions over if they have redis installed

Reply:
@arkamar Would you mind elaborating? I'm not sure how to handle this yet, tbh. But not thought about it much either (been very busy but remembered I ought to review this PR...)

Reply:
My point of view is that even if it is meant as a drop-in replacement it can run independently next to redis or any other fork I am aware of. They don't seem to have file collisions, all binaries and config files have different name, data are placed in different directories as well, e.g. `/var/lib/redict` in this case. Moreover, all forks are now backward compatible but not forward compatible which could possibly lead to an unexpected behavior if one engine is executed with data created by other engine. Having different user & group for each fork will minimize space for data corruption.

So, I think we should add new `acct-{group,user}/redict`, see [Adding `acct-{group,user}/packages` as a proxied maintainer](https://wiki.gentoo.org/wiki/Proxied_Maintainer_FAQ#Adding_acct-.7Bgroup.2Cuser.7D.2Fpackages_as_a_proxied_maintainer), if we want to add this package.

Reply:
Okay, I find that persuasive. Agreed, thanks.

Reply:
added a redict user and group!

Reply:
added a summary on the acct-user/redict commit